build fails because of src/uitl.h

gcc (6.1.1) fails with:

    In file included from src/util.cc:25:0:
    src/util.h:55:6: error: ‘vector’ in namespace ‘std’ does not name a template type
        std::vector<std::string> split(const std::string &text, char sep);
             ^~~~~~
    src/util.cc:32:6: error: ‘vector’ in namespace ‘std’ does not name a template type
        std::vector<std::string> split(const std::string &text, char sep)
             ^~~~~~

--- a/src/util.h
+++ b/src/util.h
@@ -21,6 +21,7 @@
 
 #include <algorithm>
 #include <sstream>
+#include <vector>
 
 
 /**

